tests/test_snql_api.py

```diff
@@ -77,6 +77,26 @@ def setup_teardown(self, clickhouse_db: None, redis_db: None) -> None:
             [get_raw_transaction()],
         )
 
+    def test_raw_tags_hash(self) -> None:
+        query = """MATCH (generic_metrics_counters) SELECT countIf(value, equals(metric_id, 87269613)) AS `count(g:custom/gauge@none)` WHERE org_id = 447951 AND project_id IN array(5572016) AND timestamp >= toDateTime('2023-10-25T08:00:00') AND timestamp < toDateTime('2023-11-08T12:00:00') AND tags_raw[9223372036854776010] = 'dev' AND metric_id IN array(87269613) LIMIT 21 OFFSET 0 GRANULARITY 14400"""
```

Comment:
Small note, should that MRI (`g:custom/gauge@none`) be a counter metric instead?

Reply:
it doesn't really matter, first of all it's just an alias. Also I should just remove that because what I really wanted was for CI to run with the distributed tables and actually hit the table and make sure I applied the migration correctly